java/core/src/java/org/apache/orc/Reader.java

```diff
@@ -158,6 +158,16 @@
    */
   OrcFile.WriterVersion getWriterVersion();
 
+  /**
+   * Get the implementation and version of the software that wrote the file.
+   * It defaults to "ORC Java" for old files. For current files, we include the
+   * version also.
+   * @since 1.5.13
```

Comment:
Thank you for adding `since` info, @omalley .
BTW, do we need to backport this new feature to `branch-1.5`?

Reply:
That was my intention.